lib/form_data.js

```diff
@@ -137,11 +137,11 @@ FormData.prototype._trackLength = function(header, value, options) {
           });
         }
 
-      // or http response
+        // or http response
       } else if (value.hasOwnProperty('httpVersion')) {
         next(null, +value.headers['content-length']);
 
-      // or request stream http://github.com/mikeal/request
+        // or request stream http://github.com/mikeal/request
```

Comment:
Do you mind to clean up unnecessary changes like this?
And add tests for the use case?

Thank you.